content/recipes/terminus.md

```diff
@@ -163,125 +186,60 @@ The next thing we need to do is registering the health indicator as a provider.
 @@filename(app.module)
 import { Module } from '@nestjs/common';
 import { TerminusModule } from '@nestjs/terminus';
-import { TerminusOptionsService } from './terminus-options.service';
 import { DogHealthIndicator } from './dog.health';
 
 @Module({
-  imports: [
-    TerminusModule.forRootAsync({
-      imports: [ApplicationModule],
-      useClass: TerminusOptionsService,
-    }),
-  ],
-  providers: [DogHealthIndicator],
-  exports: [DogHealthIndicator],
+  controllers: [HealthController],
+  imports: [TerminusModule],
+  providers: [DogHealthIndicator]
 })
-export class ApplicationModule { }
+export class AppModule { }
 ```
 
-> info **Hint** In a real world application the `DogHealthIndicator` should be provided in a separate module, for example `DogsModule`, which then will be imported by the `ApplicationModule`. But keep in mind to add the `DogHealthIndicator` to the `exports` array of the `DogModule` and add the `DogModule` in `imports` array of the `TerminusModule.forRootAsync()` parameter object.
+> info **Hint** In a real-world application the `DogHealthIndicator` should be provided in a separate module, for example, `DogModule`, which then will be imported by the `AppModule`.
 
-The last required thing to do is to add the now available health indicator in the required health check endpoint. For that we go back to our `TerminusOptionsService` and implement it to the `/health` endpoint.
+The last required step we need to do is to add the now available health indicator in the required health check endpoint. For that, we go back to our `HealthController` and add it to our `check` function.
```

Comment:
The last required step we need to do is
->
The last required step is

Reply:
Resolved, thanks <3